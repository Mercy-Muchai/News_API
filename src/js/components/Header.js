import React from 'react';
import ReactDOM from 'react-dom';

export default class Logo extends React.Component {
  render() {
    return (
      <div>
        <img src={require("../newsicon.png")} />
        <h1>Top News</h1>
      </div>
    )
  }
}
