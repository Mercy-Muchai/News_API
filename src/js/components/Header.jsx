import React from 'react';
import Login from './Login.jsx';

export default class Header extends React.Component {
  render() {
    return (
        <nav className="navbar navbar-default navbar-fixed-top">
          <div className="navbar-brand">
            <a className="header">
              <span style={{ display: "inline-block" }}>Get News
              </span>
            </a>
          </div>
          <ul className="nav navbar-nav pull-right">
            <li className="active"><Login /></li>
          </ul>
        </nav>
        )
  }
}
