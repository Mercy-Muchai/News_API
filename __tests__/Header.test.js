import React from 'react';
import { mount } from 'enzyme';
import shallowToJSON from 'enzyme-to-json';
import Header from '../src/js/components/Header';

describe('Header', () => {
  it('Should render a navbar', () => {
    const homes = shallow(<Header/>);
    const tree = shallowToJSON();
    expect(tree).toMatchSnapshot();
  });
});


