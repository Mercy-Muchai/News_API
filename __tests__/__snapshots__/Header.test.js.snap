// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should render a navbar 1`] = `undefined`;
